tests/integration/api/test_api.py

```diff
@@ -649,6 +649,51 @@ def test_downtime(self):
             "Downtime", downtime["id"], retry_condition=lambda r: r["disabled"] is False
         )
 
+    def test_downtime_cancel_by_scope(self):
+        scope_one = "test:integration_one"
+        scope_two = "test:integration_two"
+        start = int(time.time())
+
+        # Create downtime with scope_one
+        end = start + 1000
+        downtime_one = dog.Downtime.create(scope=scope_one, start=start, end=end)
```

Comment:
Thinking about subsequent test runs here. Would `creating` a downtime that already exists and is disabled just flip the `disabled` flag back to `False`?

Reply:
Ahh no, it will create a separate downtime with a different `downtime["id"]`.

Reply:
Sorry, to clarify: it creates a new one because the `start` and `end` will be different depending on the test execution - so you schedule it for a different time period.

Reply:
Ahh ok that sounds good to me.